### ISSUE TITLE
Handle old Firefoxes

### ISSUE THREAD

**Issue by johngruen** [mozilla/testpilot#671]
Add-on wouldn't install on 36. We should catch this on the landing page and show the same template we show for non firefox with slightly different wording.

**Comment by meandavejustice** [comment]
``` js
const UAString = navigator.userAgent.toLowerCase();
const idx = UAString.indexOf('firefox');
const version = parseFloat(UAString.slice(idx + 'firefox'.length + 1), 0);
```

**Comment by johngruen** [comment]
@meandavejustice IIRC fennec has `fennec` in in the UA & ios has `fxios`

**Comment by pdehaan** [comment]
![image](https://cloud.githubusercontent.com/assets/557895/14921015/ccfcef30-0de4-11e6-8bed-8c49d171fea7.png)

![screenshot_2016-04-29-10-20-35](https://cloud.githubusercontent.com/assets/557895/14923976/0e9713fc-0df5-11e6-8277-78ef126218f8.png)

**Comment by meandavejustice** [comment]
@johngruen I was gunna add a different snippet for the lil firefoxs issue

**Comment by johngruen** [comment]
#736

**Comment by johngruen** [comment]
Being fixed in #1310
